Benqi-Agents/Large-Delegate-Votes-Change/README.md

```diff
@@ -0,0 +1,37 @@
+# Delegate Votes Monitor Agent
+
+## Description
+
+This agent detects large (>= 30%) increments in a delegate account's QI token
+vote balance by monitoring `DelegateVotesChanged` events emitted by the token
+contract.
+
+## Supported Chains
+
+- Avalanche
+
+## Alerts
+
+- BENQI-1-1
+  - Fired when a delegate account's vote balance increases by 30% or more.
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - Metadata contains:
+    - `delegate`: Address of the related delegate
+    - `previousBalance`: Previous delegate vote balance
+    - `newBalance`: Current delegate vote balance
+
+## Test Data
+
+The agent behaviour can be verified with the following transactions:
```

Comment:
If all the txs below generate findings, it would be better to mention that. I suggest ,if possible, adding some negative cases where no finding is returned.

Reply:
I removed some positive cases (they keep being positive, I just thought it was too much cases) and added some negative ones.